test/integration/cli/test_launch.py

```diff
@@ -18,7 +18,7 @@
 # ##### #
 def test_local_launch(tt_cmd, tmpdir):
     module = "version"
-    cmd = [tt_cmd, module, "-L", tmpdir]
+    cmd = [tt_cmd, "-L", tmpdir, module]
```

Comment:
Please check that `--help` option after some sub-command, still prints this sub-command help.

Reply:
Add tests